packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_model.dart

```diff
@@ -24,13 +24,62 @@ enum PlatformOS {
   final String description;
 }
 
-// TODO(hangyujin): Handle more domain error cases.
 enum DomainError {
```

Comment:
general suggestion: it would be great if there were docs on flutter.dev that each of these errors could link to with an example of each fix.

Reply:
Agree, will dicuss with zoey about it

Reply:
Added a 'View developer guide' button to link to [android](https://developer.android.com/training/app-links/verify-android-applinks)  

Add more docs on flutter.dev can be a future step